client/src/common/components/BrushableHistogram/BrushableHistogram.tsx

```diff
@@ -487,14 +488,16 @@ class HistogramBrush extends React.PureComponent<BrushableHistogramProps> {
     const varIndex = schema?.annotations?.var?.index;
     if (!varIndex) return null;
```

Comment:
should we remove `if (!varIndex) return null;` since the component doesn't rely on it anymore 😄 ?